src/hatti/ona/forms.cljs

```diff
@@ -219,7 +219,10 @@
                                (filter #(contains? names (field-key %)))
                                (map #(str "☑ " (get-label % language) " "))
                                string/join))
-    (time-based? field) (-> answer js/moment (.format "ll"))
+    (time-based? field) (if (or (js/isNaN (new js/Date answer))
```

Comment:
Yeah, we are making many calls to moment that would be better made in one place because they're breaking all over the place.